### DISCUSSION TITLE
Using two reference files for two conditions on one flowcell

### DISCUSSION THREAD

**Discussion by petzel96** [LooseLab/readfish#311] [General]
Is it somehow possible to use multiple ref. files for different conditions on one flow cell?
We are trying to compare new polished haplotypes with the old reference and a side by side comparison on one flow cell would be perfect. But if you use one single reference file we fear that many of the reads might map to the wrong region since they are not that distinct.

**Comment by alexomics** [comment]
Not currently (using a single command) there isn't, but it is something that we are working towards

**Comment by mattloose** [comment]
One possible work around is to run two instances of readfish each with it's own configuration and reference. Then use the 

```--channels``` flag to specify a channel range - e.g:
```--channels 1 256```
and pass it your toml for condition 1.

Then start a second and use:
```--channels 257 512```
and pass it your toml for condition 2.

This should work - but is untested and you might want to try it with playback first. 

So run two standalone instances pointed at the same flowcell.

@alexomics may wish to comment further.

There will be a duplication of messages sent to MinKNOW and might get a bit messy.... but until we establish it properly this is a workaround.

**Comment by alexomics** [comment]
Yep, to add:

- The TOML files should be the same apart from the `reference` and `targets` fields
- Make sure that the logs are named differently
- This will double readfish's footprint and hasn't been tested

If you do try this, please let us know how it goes!
